data/json/terrain.json

```diff
@@ -10690,5 +10690,77 @@
       "ter_set": "t_null",
       "items": [ { "item": "material_soil", "count": [ 8, 20 ] }, { "item": "adobe_brick", "count": [ 2, 6 ] } ]
     }
+  },
+  {
+    "id": "t_leanto",
+    "type": "terrain",
+    "name": "pine lean-to",
+    "symbol": ";",
+    "color": [ "brown" ],
+    "move_cost": 2,
+    "bash": {
+      "str_min": 40,
+      "str_max": 90,
+      "ter_set": "t_dirt",
+      "sound": "crunch!",
+      "sound_fail": "whack!",
+      "items": [
+        { "item": "stick", "count": [ 2, 7 ] },
+        { "item": "splinter", "count": [ 8, 20 ] },
+        { "item": "pine_bough", "count": [ 0, 2 ] }
+      ]
+    },
+    "flags": [ "TRANSPARENT", "CONTAINER", "FLAMMABLE_ASH", "THIN_OBSTACLE", "REDUCE_SCENT", "INDOORS", "MOUNTABLE" ]
+  },
+  {
+    "id": "t_tarptent",
+    "type": "terrain",
+    "name": "tarp lean-to",
+    "symbol": ";",
+    "color": [ "light_blue" ],
+    "move_cost": 2,
+    "bash": {
+      "str_min": 6,
+      "str_max": 12,
+      "ter_set": "t_dirt",
+      "sound": "crash!",
+      "sound_fail": "whack!",
+      "items": [
+        { "item": "stick", "count": [ 1, 2 ] },
+        { "item": "splinter", "count": [ 1, 4 ] },
+        { "item": "plastic_chunk", "count": [ 4, 8 ] }
+      ]
+    },
+    "deconstruct": {
+      "ter_set": "t_dirt",
+      "items": [ { "item": "pointy_stick", "count": 4 }, { "item": "string_6", "count": 4 }, { "item": "tarp", "count": 1 } ]
+    },
+    "flags": [ "TRANSPARENT", "FLAMMABLE", "THIN_OBSTACLE", "INDOORS", "MOUNTABLE", "EASY_DECONSTRUCT" ]
+  },
+  {
+    "id": "t_raincatcher",
```

Comment:
This thing should be furniture, so that the underlying terrain type is preserved when you take it down